src/mesh_operations.cpp

```diff
@@ -243,14 +243,23 @@ Mesh* createMeshFromBinary(const char *buffer, std::size_t size, const Eigen::Ve
   const aiScene* scene = importer.ReadFileFromMemory(reinterpret_cast<const void*>(buffer), size,
                                                      aiProcess_Triangulate            |
                                                      aiProcess_JoinIdenticalVertices  |
-                                                     aiProcess_SortByPType            |
-                                                     aiProcess_OptimizeGraph          |
-                                                     aiProcess_OptimizeMeshes,
```

Comment:
They're not removed, just moved down later. Those two optimizations heavily optimize the imported meshes and scene graph. One of the results was that the root node transformation got flattened into child nodes. That made clearing the root node transformation a NOP, but clearing the child node transformations is not an option since it would modify the mesh.

But the optimizations are still performed after clearing the root node transform here: https://github.com/ros-planning/geometric_shapes/pull/52/files#diff-f5ec0f576aaf3b44a4c42c739e51980dR260